src/qt/overviewpage.cpp

```diff
@@ -428,8 +428,8 @@ void OverviewPage::updatePrivateSendProgress()
 
     QString strToolPip = ("<b>" + tr("Overall progress") + ": %1%</b><br/>" +
                           tr("Denominated") + ": %2%<br/>" +
-                          tr("Mixed") + ": %3%<br/>" +
-                          tr("Anonymized") + ": %4%<br/>" +
+                          tr("Partially mixed") + ": %3%<br/>" +
+                          tr("Ready") + ": %4%<br/>" +
```

Comment:
```suggestion
                          tr("Mixed") + ": %4%<br/>" +
```
Idk about this one... maybe "ready" is okay, it just sounds a little weird